### PR TITLE
Auto Save Freeze, Cryo-Tube Fix 

### PR DESCRIPTION
**Pull request by SirBloodBuske**
Was having issues with my branch, However the PR is back!
image
image
image
image

### CODE REVIEW

**Comment by Kurgis** [comment]
Does this also freeze ZAS, and mob processes during Save Freeze?

**Comment by SirBloodBuske** [comment]
> Does this also freeze ZAS, and mob processes during Save Freeze?

From what I have tested yes.

**Comment by ingles98** [comment]
This will need intense review on my part. Most of the code is working around the same logic i've seen and tested myself on your older PR on the subject. Overall it looks good and ill be commenting on what should be changed and whether its accepted.
Ill take your word that you have tested it thoroughly.

**Comment by SirBloodBuske** [comment]
> This will need intense review on my part. Most of the code is working around the same logic i've seen and tested myself on your older PR on the subject. Overall it looks good and ill be commenting on what should be changed and whether its accepted.
> Ill take your word that you have tested it thoroughly.

Needs to be tested large scale, To see if the new system breaks anything.

File: code/game/machinery/cryo.dm [hunk 1]
```diff
@@ -20,59 +20,33 @@
...
-	for(var/obj/machinery/atmospherics/target in get_step(src,node_connect))
-		if(target.initialize_directions & get_dir(target,src))
-			node = target
-			break
 
 /obj/machinery/atmospherics/unary/cryo_cell/Destroy()
 	var/turf/T = loc
 	T.contents += contents
-	var/obj/item/weapon/reagent_containers/glass/B = beaker
 	if(beaker)
```
Comment (ingles98) [inline code/game/machinery/cryo.dm]: where does beaker exist?

**Comment by ingles98** [comment]
look i do not understand why do you have a cryo change in this PR and it changes so much i cant even understand it

File: code/game/machinery/cryo.dm (hunk 1 above)
Comment (SirBloodBuske) [reply code/game/machinery/cryo.dm]: It exists at the top of the code

**Comment by ingles98** [comment]
All these changes should be remade since the load/save mechanism as changed greatly. Please make sure it does not have anything else other than the Auto Save freeze featured changes, as i not only see cryo-tube stuff, it also has newspaper and xenobiological changes, it makes no sense. Overall the actual objective of this PR was met, it needs to be re-made upon the current mechanism code.

**Pull request closed without merging**